en/docs/chapter_sorting/bubble_sort.md

```diff
@@ -27,12 +27,12 @@ As shown in the figure below, the bubbling process can be simulated using elemen
 
 ## Algorithm process
 
-Assuming the length of the array is $n$, the steps of bubble sort are shown in the figure below.
+Assume the array has length $n$. The steps of bubble sort are shown in the figure below:
 
-1. First, perform a "bubble" on $n$ elements, **swapping the largest element to its correct position**.
-2. Next, perform a "bubble" on the remaining $n - 1$ elements, **swapping the second largest element to its correct position**.
-3. Similarly, after $n - 1$ rounds of "bubbling," **the top $n - 1$ largest elements will be swapped to their correct positions**.
-4. The only remaining element is necessarily the smallest and does not require sorting, thus the array sorting is complete.
+1. First, perform one “bubble” pass on all $n$ elements, **swapping the largest element to its correct position**.
```

Comment:
1.I think "bubble" is a technical term which could be used directly. How about update to this: perform "bubble" on n elements.
2.This sentence has grammar error. "...elements, and swap the xxx."